src/components/Alert/Alert.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { variant } from 'styled-system';
+import theme from '../../theme';
+import { get } from '../../utils/utils';
+import { COMMON, COLOR } from '../../utils/constants';
+
+const Alert = styled.div`
+  // text
+  ${get('typography.alert')};
+
+  // border
+  border-radius: 4px;
+
+  // color
+  ${variant({ scale: 'alert' })}
+
+  // display
+  display: flex;
+
+  // align
+  vertical-align: middle;
+  align-items: center;
+
+  // size
+  padding: 0px 18px;
+  height: 55px;
+
+  svg {
+    margin-right: 18px;
```

Comment:
should we include `float: left` or any margin on the left??

Reply:
we could do float left, but the padding should push the icon to the right by the appropriate amount